sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -65,18 +65,18 @@
         new ClosedChannelException(), RntbdClientChannelPool.class, "acquire");
 
     private static final IllegalStateException POOL_CLOSED_ON_ACQUIRE = ThrowableUtil.unknownStackTrace(
-        new StacklessIllegalStateException("service endpoint was closed while acquiring a channel"),
+        new ChannelAcquisitionException("service endpoint was closed while acquiring a channel"),
         RntbdClientChannelPool.class, "acquire");
 
     private static final IllegalStateException POOL_CLOSED_ON_RELEASE = ThrowableUtil.unknownStackTrace(
-        new StacklessIllegalStateException("service endpoint was closed while releasing a channel"),
+        new ChannelAcquisitionException("service endpoint was closed while releasing a channel"),
```

Comment:
Is release also related to Acuisition?

Reply:
It's what happens immediately after a channel is acquired and the request is dispatched to the channel. We've got a promise that the write will occur, but have not yet received a TCP ACK.